binary-store.cabal

```diff
@@ -22,7 +24,7 @@ library
                , bzlib
                , deepseq
   default-language: Haskell2010
-  ghc-options: -O2 -Wall
+  ghc-options: -O2 -Wall -Werror
```

Comment:
Don't we need to add these flags too?
- -Wincomplete-uni-patterns
- -Wincomplete-record-updates

Reply:
Was that a hard requirement? OK. I can add them.

Reply:
Done here: https://github.com/plow-technologies/binary-store/pull/2/commits/6c6242305e987c86e11886a0ae3af568b40c947e